libqfieldsync/offline_converter.py

```diff
@@ -85,6 +85,8 @@ def __init__(
         self,
         project: QgsProject,
         export_folder: str,
+        export_filename: str,
+        export_title: str,
```

Comment:
```suggestion
        export_filename: str,
        export_title: str,
```

These should be optional because we want to be backward compatible with QFieldCloud.